packages/drivers/odsp-driver/src/createFile/createFile.ts

```diff
@@ -102,6 +104,20 @@ export async function createNewFluidFile(
 	fileEntry.docId = odspResolvedUrl.hashedDocumentId;
 	fileEntry.resolvedUrl = odspResolvedUrl;
 
+	odspResolvedUrl.context = resolvedUrl?.context;
+	odspResolvedUrl.appName = resolvedUrl?.appName;
+
```

Comment:
Also transfer the container package name?

Reply:
Done